### ISSUE TITLE
Mozilla Firefox Extension

### ISSUE THREAD

**Issue by etbusch** [abzubarev/web-developer-form-filler-ext#20]
At first glance this looks to be compatible with Firefox. Would you like to submit it to the extension store, or mind if I did on your behalf?

**Comment by AliOko** [comment]
Is there a chance to Firefox extension?

**Comment by etbusch** [comment]
For those interested, I've forked this extension and updated it to work with Firefox.
Get it here: https://addons.mozilla.org/en-US/firefox/addon/web-dev-form-filler/

**Comment by abzubarev** [comment]
That's too late, but I would prefer a pull request

**Comment by etbusch** [comment]
Yeah, I was hoping to do so, but this project looked abandoned at the time I made the fork. 

See https://github.com/etbusch/web-developer-form-filler-ext/issues/17 for details on what I've done on the FF version of this extension.
